commons/src/main/java/info/metadude/android/eventfahrplan/commons/temporal/Moment.kt

```diff
@@ -108,12 +109,29 @@ class Moment private constructor(private val time: Instant) {
     override fun toString(): String = time.toString()
 
     companion object {
+
+        /**
+         * 1 minute = 60 seconds
+         */
+        private const val SECONDS_OF_ONE_MINUTE: Int = 60
+
         /**
          * Creates a time zone neutral [Moment] instance of current system clock.
          */
         @JvmStatic
         fun now() = Moment(Instant.now())
 
+        /**
+         * Returns the amount of minutes between the UTC and the system default time zone.
+         */
+        @JvmStatic
+        fun getSystemOffsetMinutes(): Int {
+            val dateTime = LocalDateTime.now()
```

Comment:
If I'm using this code on Oct 24th 2020 to look for events happening the next day, it will return incorrect offset (120). Maybe it is okay if only used for scrolling, but to give correct result, "dateTime" must be a parameter to this method instead of using "now".